src/components/Inputs/InputSelect.js

```diff
@@ -0,0 +1,25 @@
+export const InputSelect = props => {
+    const {label, value, error, showError } = props
+    return (
+      <div className="form-group">
+        <label>{label}</label>
+        <select 
+          value={value} 
+          onChange={e => props.onChange(e.target.value)}
+          className={`form-control ${error && showError ? 'is-invalid' : ''}`}>
+            {props.options.map(option => 

```

Comment:
remove props

Reply:
remove only from line 10 or line 8 to and save onChange in destructor?